clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/ops/EnableServiceAtomicOperation.java

```diff
@@ -52,16 +60,85 @@ private static Task getTask() {
 
   @Override
   public Void operate(List priorOutputs) {
-    getTask().updateStatus(BASE_PHASE, "Initializing Enable Amazon ECS Server Group Operation...");
+    updateTaskStatus("Initializing Enable Amazon ECS Server Group Operation...");
+    enableService();
+    return null;
+  }
 
-    AmazonCredentials credentials = (AmazonCredentials) accountCredentialsProvider.getCredentials(description.getCredentialAccount());
-    AmazonECS ecs = amazonClientProvider.getAmazonEcs(description.getCredentialAccount(), credentials.getCredentialsProvider(), description.getRegion());
-    String clusterName = containerInformationService.getClusterName(description.getServerGroupName(), description.getCredentialAccount(), description.getRegion());
+  private void enableService() {
```

Comment:
Same as in disableService - please add arguments to this method, and break it down into smaller pieces

Reply:
Same as disable service, I would prefer to delay refactoring for testability until we have a testing strategy. 

Same refactoring done for the cluster name. Unsure of how to further break it down.